### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by taz4et** [erew123/alltalk_tts#207] [Q&A]
Hi, 
How can I contact the maintainer via Telegram?

edit:
https://t.me/HoseDemure

**Comment by RenNagasaki** [comment]
If he wants that to be possible he may send you a DM, But why would you want that though? 🤣

**Comment by taz4et** [comment]
)) I never used GitHub as a social netowrk before, and have no idea how to send a DM here

**Comment by RenNagasaki** [comment]
Just found out that you cant 🤣
Normally all communication gets handled via Issues.

**Reply by taz4et** [reply]
This is not about issues, but rather a feature request that I wanted to discuss more effectively :)

**Reply by RenNagasaki** [reply]
Ahh, I see. Then I'd recommend starting a discussion for that, and when the request is more clear, create a feature request under issues. 😁

I doubt he'll openly tell you his contact where everyone can see it.

**Reply by taz4et** [reply]
yeah, that's why I edited the first message and added my details. So if he needs more Ko-fi donors, I'm here.

**Comment by erew123** [comment]
Hi @arasaahov 

This is something you don't want to mention/talk about here? Do you have Discord at all? I have an account there and can share my ID if you like.

Thanks
